### PR TITLE
Remove xfail for COTS test sequence n.8

### PR DESCRIPTION
**Pull request by mbounabi**
the associated refs:
https://github.com/CanalTP/artemis_references/pull/157

### CODE REVIEW

File: artemis/tests/guichet_unique_test.py [hunk 1]
```diff
@@ -1040,6 +967,81 @@ def test_kirin_cots_sequence_05(self):
...
+                     max_nb_transfers="0",
+                     data_freshness="realtime")
+
+        self.journey(_from="stop_area:OCE:SA:87185009",
+                     to="stop_area:OCE:SA:87751008",
+                     datetime="20121120T163000",
+                     max_nb_transfers="0",
+                     data_freshness="base_schedule")
+
+        self.journey(_from="stop_area:OCE:SA:80110684",
```
Comment (pbougue) [inline artemis/tests/guichet_unique_test.py]: Shouldn't we test that also after 1st and 3rd feed?

File: artemis/tests/guichet_unique_test.py [hunk 2]
```diff
@@ -1040,6 +967,81 @@ def test_kirin_cots_sequence_05(self):
...
+                     max_nb_transfers="0",
+                     data_freshness="base_schedule")
+
+        """
+        2. Add new stop
+        Requested departure: 2012/11/20 16:30:00
+        From: gare de Vesoul (Vesoul)
+        To: gare de Marseille-St-Charles (Marseille)
+
+        Before detour, no solution can be found without transfer
```
Comment (pbougue) [inline artemis/tests/guichet_unique_test.py]: minor: if I understand correctly, it's an add (more than a detour in a SNCF way)

File: artemis/tests/guichet_unique_test.py [hunk 3]
```diff
@@ -1040,6 +967,81 @@ def test_kirin_cots_sequence_05(self):
...
+        Requested departure: 2012/11/20 14:00:00
+        From: gare de Bitche (Bitche) that replaces gare de Frankfurt-am-Main-Hbf
+        To: gare de Marseille-St-Charles (Marseille)
+
+        Before detour, no solution can be found without transfer
+        After detour, the train travels from 14:20:00 to 22:11:00
+        """
+        self.send_and_wait('trip_seq8_01_detour_start_delay_25.json')
+
+        self.journey(_from="stop_area:OCE:SA:87193821",
```
Comment (pbougue) [inline artemis/tests/guichet_unique_test.py]: Should we keep on testing this after 2nd and 3rd feed?

Comment (mbounabi) [reply artemis/tests/guichet_unique_test.py]: I'll add the test at the end

File: artemis/tests/guichet_unique_test.py [hunk 4]
```diff
@@ -1040,6 +967,81 @@ def test_kirin_cots_sequence_05(self):
...
+                     data_freshness="realtime")
+
+        """
+        3. Add more delay
+        Requested departure: 2012/11/20 16:30:00
+        From: gare de Vesoul (Vesoul)
+        To: gare de Marseille-St-Charles (Marseille)
+
+        Before detour, no solution can be found without transfer
+        After detour, the train travels from 16:52:00 to 22:26:00
```
Comment (mbounabi) [inline artemis/tests/guichet_unique_test.py]: 'delay' not 'detour'

**Pull request merged**